File: FE/src/app/services/player.service.ts
import { HttpClient } from '@angular/common/http';
import { Injectable } from '@angular/core';

@Injectable({
  providedIn: 'root'
})
export class PlayerService {
  playerURL : string = "http://localhost:8080/api/players"
  constructor(private http:HttpClient) {}

  getAllPlayers(){}
  getPlayerById(){}
  addPlayer(){}
  updatePlayer(){}
  deletePlayerById(){}
}

File: BE/src/main/java/com/example/crocojuinspring/summersport/services/PlayerService.java
package com.example.crocojuinspring.summersport.services;

import java.util.List;

import com.example.crocojuinspring.summersport.models.PlayerModel;


public interface PlayerService {

	public PlayerModel addPlayer(PlayerModel m);
	
	public List<PlayerModel> getAllPlayers();
	
	public PlayerModel getPlayerById(Long id);
	
	public void deletePlayerById(long id);
	
	public PlayerModel updatePlayer(PlayerModel m);
	
}
